### PR TITLE
Issue/497 logging all communication

### PR DESCRIPTION
**Pull request by maciekpe**
Logging/filltering/obfuscation for req/res  as json/xml at both sides of MS: - spring controllers, rest client

### CODE REVIEW

File: micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/web/logging/RequestLoggingConfiguration.java [hunk 1]
```diff
@@ -1,10 +1,14 @@
 package com.ofg.infrastructure.web.logging;
 
+import com.ofg.infrastructure.web.logging.config.LogsConfig;
+import com.ofg.infrastructure.web.logging.obfuscation.*;
```
Comment (mariuszs) [inline micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/web/logging/RequestLoggingConfiguration.java]: please extend

File: micro-infra-spring-base/src/main/java/com/ofg/infrastructure/web/logging/config/LogsConfigElement.java [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+import com.google.common.collect.Iterables;
+import org.apache.commons.lang.StringUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class LogsConfigElement {
+
+    String skipAll = Boolean.FALSE.toString();
```
Comment (mariuszs) [inline micro-infra-spring-base/src/main/java/com/ofg/infrastructure/web/logging/config/LogsConfigElement.java]: private

**Comment by mariuszs** [comment]
:+1:

**Comment by marcingrzejszczak** [comment]
@maciekpe - can you add some documentation to the WIKI ?

**Pull request merged**